### В университет на факультеты A и B поступило равное количество студентов, а на факультет C студентов поступило столько же, сколько на A и B вместе. Вероятность того, что студент факультета A сдаст первую сессию, равна 0.8. Для студента факультета B эта вероятность равна 0.7, а для студента факультета C - 0.9. Студент сдал первую сессию. Какова вероятность, что он учится: a). на факультете A б). на факультете B в). на факультете C?

Какова вероятность, что он учится:

a) на факультете A
б) на факультете B
в) на факультете C?

А - студент сдал первую сессию

P(B1) = P(B2) = 1/4 P(B3) = 2/4 = 1/2

P(A) = P(B1) * P(A|B1) + P(B2) * P(A|B2) + P(B3) * P(A|B3) - полная вероятность

После нахождения полной вероятности определим, что это был определенный студент при помощи формулы: P(Bn) * P(A|Bn) / P(A)

In [1]:
A = 1/4 * 0.8 + 1/4 * 0.7 + 1/2 * 0.9
first = (1/4 * 0.8) / A 
second = (1/4 * 0.7) / A 
third = (1/2 * 0.9) / A 

first, second, third

(0.24242424242424246, 0.21212121212121213, 0.5454545454545455)

Ответ: Вероятность того, что это был студент из факультета:

А = 24.24%

B = 21.21%

C = 54.55%